draft-ftbs-rats-msg-wrap.md

```diff
@@ -70,6 +75,8 @@ message(s) are carried in the value.
 The other format wraps the value in a CBOR byte string and prepends a
 CBOR tag to convey the type information.
 
+An associated CBOR tag, as well as JWT and CWT claims are also defined to allow embedding these formats into CBOR-based protocols and web tokens.
```

Comment:
```suggestion
This document defines a corresponding CBOR tag as well as JWT and CWT claims allow embedding these formats into CBOR-based protocols and web tokens.
```

Reply:
Maybe:
formats -> data structures?

Reply:
I have updated your suggestion:
* to fix some mismatched grammar, and
* change "formats" into "wrapped conceptual messages"

Reply:
Maybe say "...into CBOR-based protocols and structures, e.g., web tokens."

Reply:
:sparkles: lgtm